src/utils/resolveRenderArgs.js

```diff
@@ -1,5 +1,27 @@
 import HttpError from '../HttpError';
 
+function foldElements(elementsRaw, routeIndices) {
```

Comment:
Is this the right place for this logic? I didn't want to change the signature for `<ElementsRenderer>`.